docs_src/getting-started/quick-start/index.md

```diff
@@ -1,227 +1,255 @@
-This guide will get EdgeX up and running on your development machine in as little as 5 minutes. We will will skip over lengthy descriptions for now, you can read up on those later. The goal here is to get you a working IoT Edge stack, from device to cloud, as simply as possible.
-
+This guide will get EdgeX up and running on your development machine in as little as 5 minutes. We will skip over lengthy descriptions for now, you can read up on those later. The goal here is to get you a working IoT Edge stack, from device to cloud, as simply as possible.
 
 ## Setup
-
-
 The fastest way to start running EdgeX is by using our pre-built Docker images. To use them you'll need to install the following:
 
 * Docker <https://docs.docker.com/install/>
 * Docker Compose <https://docs.docker.com/compose/install/>
 
-
-
 ## Running EdgeX
+Once you have Docker and Docker Compose installed, you need the file for downloading and running the EdgeX Foundry docker containers. Download the latest [`docker-compose` file](https://github.com/edgexfoundry/developer-scripts/blob/master/releases/geneva/compose-files/docker-compose-geneva-redis-no-secty.yml) and save this as `docker-compose.yml` in your local directory. This file contains everything you need to deploy EdgeX with docker.  
```

Comment:
i think we can condense this down. Instead of walking them through each step -- how about we give a single command do download and run the compose?

Reply:
Once you have docker and docker compose installed, execute the following to download and run EdgeX. 
```
wget https://raw.githubusercontent.com/edgexfoundry/developer-scripts/master/releases/geneva/compose-files/docker-compose-geneva-redis-no-secty.yml -o docker-compose.yml; docker-compose up
```
Boom done.

Reply:
yep - cleaning up now.

Reply:
done and used tabs for x86 v ARM